framework/src/actions/AddVariableAction.C

```diff
@@ -235,8 +235,8 @@ AddVariableAction::createInitialConditionAction()
 std::string
 AddVariableAction::determineType(const FEType & fe_type, unsigned int components, bool is_fv)
 {
-  mooseDeprecated("AddVariableAction::determineType() is deprecated. Use "
-                  "AddVariableAction::variableType() instead.");
+  ::mooseDeprecated("AddVariableAction::determineType() is deprecated. Use "
```

Comment:
why is this necessary? This is a inherited class method right?

Reply:
we re in a static function so we cant use a non-static member function